### ISSUE TITLE
Test Web: Bad rendering of buttons in custom dialog

### ISSUE THREAD

**Issue by octref** [microsoft/vscode#83557]
Testing #83296:

Version: 1.40.0-insider
Commit: b7b21844e760e632f39bedf651da2bc93f54b0cc
Date: 2019-10-29T06:38:51.389Z
Browser: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:70.0) Gecko/20100101 Firefox/70.0

Run the command Help -> About:

![image](https://user-images.githubusercontent.com/4033249/67791535-d4fb6780-fa34-11e9-970b-0b6d45de6cb2.png)

**Comment by bpasero** [comment]
@sbatten while looking into https://github.com/microsoft/vscode/issues/83573 I noticed how we use `max-width: fit-content` for buttons in dialogs and notifications. I think there is a bug in FF that results in what we see here, but I also really do not remember why we used max-width in the first place. We probably want to overflow: ellipsis when there are many buttons, but I can still see that when switching to `width: fit-content`, which I think is exactly the intent (give buttons enough width to show their text). 

I hope this is not causing regressions, let's see how it goes. It is changed for electron+web via https://github.com/microsoft/vscode/commit/557cf012ca0d1e25366af58185de5221d47b39a3
